reference/supportability.md

```diff
@@ -156,4 +150,43 @@ The archive files generated by the dump command are stored in the specified outp
 | :--- | :--- | :--- |
 | ./ | 	Support-tool | support_tool_logs.log |
 </details>
+
+## Does the supportability tool expose sensitive data?
+
+The supportability tool generates support bundles, which are used for debugging purposes. These bundles are created in response to the user's invocation of the tool and can be transmitted only by the user.
+Below is the information collected by the supportability tool that might be identified as 'sensitive' based on the organization's data protection/privacy commitments and security policies.
+**Logs:**
+The default installation of Mayastor includes the deployment of a log aggregation subsystem based on Grafana Loki. All the pods deployed in the same namespace as Mayastor and labelled with `openebs.io/logging=true` will have their logs incorporated within this centralized collector. These logs may include the following information:
+- Kubernetes(K8s) node hostnames
```

Comment:
A space needed between Kubernetes and (K8s) ?

Reply:
This comment has been addressed as part of https://github.com/openebs/mayastor-docs/pull/89/commits/e4e3aa20ff8a7a07d0f41f5a86bed1367c6d704f commit